Fix pipeline yaml

karthikjeeyar opened the pull request:
Fixed the pipeline resource creation issues
* Increased the delay between each resource creation

sbose78 commented:
build and deploy

sbose78 commented:
Your build is progress : https://console-openshift-console.apps.rgarg14.devcluster.openshift.com/k8s/ns/openshift-console/builds
 
 PR will be LIVE in https://pr-144-openshift-console.apps.rgarg14.devcluster.openshift.com

Commit:
add snapshots integration test

Commit:
Fixes access mode support for PVC creation

Signed-off-by: Kanika Murarka <kmurarka@redhat.com>

christoph-jerolimov committed:
Improve secret hostname hint

vbnrh committed:
Adds default label to snapshot class with the correct annotation

Signed-off-by: Vineet Badrinath <vbadrina@redhat.com>

JPinkney committed:
Allow cluster admins to create terminals in the openshift-terminal namespace

Modify backend proxy check to allow cluster admins if and only if their
terminal is in the openshift-terminal namespace

Only search for terminals in openshift-terminal namespace when using cluster admin

Rework terminal proxy to use isClusterAdmin for checking if the user can access the terminal

Use error status to see if namespace exists in CloudShellAdminSetup

Return false if namespace is not found in CloudShellAdminSetup instead of displaying error.
The reason we need to do this is if you set the error the terminal will flash quickly with
the error before switching to the newly created terminal.

Make subject access review request against openshift-terminal namespace

Display loading screen until admin check has finished

Signed-off-by: Josh Pinkney <joshpinkney@gmail.com>

cloudbehl committed:
Create stretch cluster with arbiter node.

Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

Commit:
i18n ocs dashboards
 - independent mode dashboard
 - internal mode dashboard
 - object service dashboard

Additional modifications:
- dropdown options format to be {name: /*the string to be displayed and used for i18n*/,id: /*a unique name of the option-key*/} on all ocs dashboards
- unit test adjustments
- i18n of empty states in core components of details-card,inventory-card,graph-empty

Namespaces added: ceph-storage-plugin, noobaa-storage-pugin

Signed-off-by: Afreen Rahman <afrahman@redhat.com>

Commit:
gherkin script for quick search to add project in topology view- ODC-2925

reginaelyse committed:
Add argo cd project-specific link to details page

andrewballantyne committed:
Devfile - Frontend

Co-Authored-By: Regina Scott <reginaelyse@gmail.com>
Co-Authored-By: Jaideep Raghunath Rao <jaideep.r97@gmail.com>

kyoto committed:
Monitoring: Add i18n for Metrics and Dashboards pages

reginaelyse committed:
renamed Application Stages to Environments

rawagner committed:
Check certificate before uploading

rawagner committed:
Use kubevirt sc config map

nemesis09 committed:
enable quick starts hints in quick start side panel

glekner committed:
add create vm flow to dev console

sahil143 committed:
add sync capability to useUserSeetings hook

a

fix unit tests

sahil143 committed:
Convert Pinned Resources local storage to user settings

fix unit tests

fix edge case

address review comments

address review comments

pcbailey committed:
Localize Virtualization nav name

pcbailey committed:
Localize the Bare Metal Hosts side nav name

debsmita1 committed:
Convert custom table column configuration from localStorage to user settings

vikram-raj committed:
add Kamelets- schema generated forms & yaml/form switching

karthikjeeyar committed:
add pipeline metrics dashboard

rottencandy committed:
update associated pipeline during edit flow

bond95 committed:
Added init state to insights plugin

rawagner committed:
i18n for advanced wizard

pcbailey committed:
Localize remaining kubevirt-plugin modals

bipuladh committed:
Changes to Capacity Metrics in OCS Persistent Storage Dashboard
 - Add Raw Capacity Card in Dashboard
 - Remove Avaiable Metric from Breakdown Card
 - Rename Capacity Breakdown Card
 - Add Info Tip to Capacity Breakdown Card

dtaylor113 committed:
Cypress Annotations Tests

jeff-phillips-18 committed:
Add i18n to topology package

makambalaji committed:
added pipeline related files

invincibleJai committed:
Standardize Capitalization knative , devconsole, pipelines, topology and console shared

andrewballantyne committed:
Devfile - Backend

Co-Authored-By: Maysun J Faisal <maysunaneek@gmail.com>
Co-Authored-By: Regina Scott <reginaelyse@gmail.com>
Co-Authored-By: Jaideep Raghunath Rao <jaideep.r97@gmail.com>

andrewballantyne committed:
Devfile - Backend (dependencies)

Co-Authored-By: Maysun J Faisal <maysunaneek@gmail.com>
Co-Authored-By: Regina Scott <reginaelyse@gmail.com>
Co-Authored-By: Jaideep Raghunath Rao <jaideep.r97@gmail.com>

Commit:
gherkin script: display TaskRuns associated with PipelineRun -ODC-3712

jeff-phillips-18 committed:
Fix sorting of nav section items

rhamilto committed:
Bug 1897354:  make CRDCard display consistent with tabs

cloudbehl committed:
setting storage class when selectedKey is set in storageClassDropDown

Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

dtaylor113 committed:
Bug 1897008: Cypress: reenable check for 'aria-hidden-focus' rule & checkA11y test for modals

openshift-merge-robot committed:
Merge pull request #7371 from metalice/bz-1902969-cannot-load-vmi-detail-page

Bug 1902969: Added a check on the value return from getOwnerReferences

openshift-merge-robot committed:
Merge pull request #7372 from afreen23/i18n-dash-statuses

i18n overview dashboard health state messages

openshift-merge-robot committed:
Merge pull request #7272 from reginapizza/jira-issue-ODC-5144

GitOps- Renamed Application Stages in nav to Environments

openshift-merge-robot committed:
Merge pull request #7311 from rawagner/vmt_support_custom

Custom support statement for templates

openshift-merge-robot committed:
Merge pull request #7315 from jeff-phillips-18/topology-i18n

Add i18n to topology package

openshift-merge-robot committed:
Merge pull request #7327 from jerolimov/odc-4755-hook

Update user settings hook to use bridge call to create ConfigMap

openshift-merge-robot committed:
Merge pull request #7169 from gouyang/fix_46_test_failure

Bug 1896302: Fix 4.6 test failures

openshift-merge-robot committed:
Merge pull request #7153 from a2batic/kms-cluster-encryption

Kms cluster encryption

openshift-merge-robot committed:
Merge pull request #7333 from yaacov/metal3-i18n-iii

Metal3 i18n form

openshift-merge-robot committed:
Merge pull request #7368 from afreen23/i18n-ocs-dashboard

i18n ocs dashboards

metalice committed:
Added isVMPage boolean to choose correct links for Nics and Disks

Signed-off-by: Matan Schatzman <matanschatzman@gmail.com>

yaacov committed:
Add missed i18n strings

cloudbehl committed:
Validation on storage & nodes install step

Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

jeff-phillips-18 committed:
Bug 1902595: Fix to show alerts in topology list view

openshift-merge-robot committed:
Merge pull request #7188 from glekner/dev-console-add-vm

Add VM Create to Dev console

openshift-merge-robot committed:
Merge pull request #7240 from sanketpathak/gherkin_provide_quick_search_to_add_to_project

gherkin script for quick search to add project in topology view- ODC-2925

akashshinde committed:
Helm chart list should not return charts if the repo is disabled

openshift-merge-robot committed:
Merge pull request #7012 from akashshinde/filter-charts

Bug 1891314: Return helm charts based on the installed kubernetes version

glekner committed:
add sc detection to vm snapshots

sg00dwin committed:
Position Overview status popovers to the top
Fix Bug 1903196

rohitkrai03 committed:
Add support for samples & snippets in pipeline builder

openshift-merge-robot committed:
Merge pull request #7335 from pcbailey/localize-virtualization-nav-name

Localize Virtualization nav name

openshift-merge-robot committed:
Merge pull request #7365 from spadgett/upgrade-node-sass

Bug 1901594: bump node-sass to 5.0.0

openshift-merge-robot committed:
Merge pull request #7383 from cloudbehl/storge-class-selection-fix

setting storage class when selectedKey is set in storageClassDropDown

openshift-merge-robot committed:
Merge pull request #7387 from metalice/bz-1903402-Nic-Disk-on-VMI-overview-should-link-to-VMI-nic-disk-page

Bug 1903402: Added isVMPage boolean to choose correct links for Nics and Disks

TheRealJon committed:
Bug 1901531: Fix runtime error in create CR form

In some cases, an uncaught exception could be thrown from dynamically generated array form fields.
This was happening when a nested array field had an explicitly set 'null' value. This is a
workaround that adds a fallback value to object and array fields to prevent this runtime error. An
issue and fix have also been opened upstream in the react-jsonschema-form repo.
https://github.com/rjsf-team/react-jsonschema-form/issues/2153
https://github.com/rjsf-team/react-jsonschema-form/pull/2154

christianvogt committed:
remove vm templates from template catalog

openshift-merge-robot committed:
Merge pull request #7388 from yaacov/missed-i18n-strings

Add missed i18n strings

openshift-merge-robot committed:
Merge pull request #7281 from nemesis09/enable-quickstart-hint

Enable quickstart hints in quickstart sidepanel

invincibleJai committed:
adds feature flag to catalog item provider

openshift-merge-robot committed:
Merge pull request #7396 from rohitkrai03/yaml-editor-sidebar

Add support for samples & snippets in pipeline builder yaml editor

gajanan-more committed:
Gherkin Scripts for ODC-4181 and ODC-3954

Commit:
KMS flow for storage class creation

Signed-off-by: Kanika Murarka <kmurarka@redhat.com>

rawagner committed:
Fix common template providers

openshift-merge-robot committed:
Merge pull request #7344 from vikram-raj/odc-5063

add Kamelets- schema generated forms & yaml/form switching

openshift-merge-robot committed:
Merge pull request #7400 from invincibleJai/fix-kubevirt-reqd-provider

adds feature flag to catalog item provider

openshift-merge-robot committed:
Merge pull request #7374 from invincibleJai/case-i18n

Standardize Capitalization knative , devconsole, pipelines, topology, console app, container security and console shared

sahil143 committed:
convert last used perspective local storage to user settings

address review comments

yaacov committed:
kubvirt i18n actions

christoph-jerolimov committed:
Add missing PropTypes, fix invalid prop to fix build

dtaylor113 committed:
added aria-label to Cancel button of Modal factory component

openshift-merge-robot committed:
Merge pull request #6882 from a2batic/access-mode2

Bug 1882057: Fixes access mode support for PVC creation

openshift-merge-robot committed:
Merge pull request #7319 from cyril-ui-developer/cannot-search-operator-by-name

Bug 1899200: - In Installed Operators page I cannot search for an Operator by it's name

openshift-merge-robot committed:
Merge pull request #7389 from cloudbehl/validations

Validation on storage & nodes install step

rebeccaalpert committed:
Added revised translations in response to Ya Dan's feedback

Commit:
gherkin_usability_enhancement_for_quick_start-ODC-4653

christoph-jerolimov committed:
Add lazy loaded snippet for default console catalog categories

Commit:
Signed-off-by: Afreen Rahman <afrahman@redhat.com>

i18n ocs day1 and day2 operations

- create storage cluster
- create backing store
- create bucket class
- add capacity
- disk replacement
- create new pool
- edit bucket class
- disk inventory page
- create OB/OBC
- details page of OB/OBC

Also i18n the lso plugin, since ocs is dependent on lso package

Signed-off-by: Afreen Rahman <afrahman@redhat.com>

jeff-phillips-18 committed:
Use extensions to add decorators to topology nodes

openshift-merge-robot committed:
Merge pull request #7336 from pcbailey/localize-bare-metal-hosts-nav-name

Localize the Bare Metal Hosts side nav name

rohitkrai03 committed:
Move all helm code into separate monorepo package

christoph-jerolimov committed:
Add workaround to support favorite namespace when loading the app

sahil143 committed:
convert last used namespace local storage to userSettings

rohitkrai03 committed:
Update i18n for helm-plugin package

openshift-merge-robot committed:
Merge pull request #7345 from karthikjeeyar/pipeline-metrics

add pipeline metrics dashboard

openshift-merge-robot committed:
Merge pull request #7346 from rottencandy/pipelineUpdate

Update associated pipeline during the edit flow

openshift-merge-robot committed:
Merge pull request #7351 from bond95/pending-state-insights

Added init state to insights plugin

openshift-merge-robot committed:
Merge pull request #7369 from akashshinde/disable-helm-chart

Helm chart list should not return helm charts if the repo is disabled.

openshift-merge-robot committed:
Merge pull request #7378 from rawagner/vmt_advanced_translate

i18n for advanced wizard

openshift-merge-robot committed:
Merge pull request #7395 from sg00dwin/overview-status-popover-bug1903196

Bug 1903196: Position Overview status popovers to display on top

openshift-merge-robot committed:
Merge pull request #7397 from TheRealJon/1901531

Bug 1901531: Fix runtime error in create CR form

openshift-merge-robot committed:
Merge pull request #7398 from christianvogt/catalog-templates

remove vm templates from template catalog

openshift-merge-robot committed:
Merge pull request #7403 from rawagner/vmt_label_obj

Fix common template providers

openshift-merge-robot committed:
Merge pull request #7404 from bipuladh/new_card

Changes to Capacity Metrics in OCS Persistent Storage Dashboard

openshift-merge-robot committed:
Merge pull request #6907 from harishsurf/accessibility_dropdown

Bug 1874968: Pressing escape key doesn't exit the dropdown list

openshift-merge-robot committed:
Merge pull request #7291 from sahil143/odc-5109

Convert Pinned Resources local storage to user settings

openshift-merge-robot committed:
Merge pull request #7392 from jeff-phillips-18/list-view-alerts

Bug 1902595: Fix to show alerts in topology list view

openshift-merge-robot committed:
Merge pull request #7145 from JPinkney/secured-namespaces

[ODC-4388] Allow cluster admins to create terminals

openshift-merge-robot committed:
Merge pull request #7288 from reginapizza/reginascott-GitOpsEnvArgoLinks

Add ArgoCD application links to GitOpsDetailsPage cards

openshift-merge-robot committed:
Merge pull request #7299 from andrewballantyne/devfile-rework

Devfile Add Flow

jcaianirh committed:
Bug 1903172: background shadow appears under datalist in column management modal

openshift-merge-robot committed:
Merge pull request #7394 from yaacov/kubevirt-i18n-actions

kubevirt i18n Actions

yaacov committed:
fix merge conflicts

bipuladh committed:
Fix error state for Raw Capacity Card
- Align Error message for capacity breakdown card to the center

Commit:
(feat): Enable flexible scaling in storage cluster deployment
 - implements https://issues.redhat.com/browse/RHSTOR-1399

Signed-off-by: Afreen Rahman <afrahman@redhat.com>

openshift-merge-robot committed:
Merge pull request #7339 from debsmita1/custom-table-US

Convert custom table column configuration from localStorage to user settings

openshift-merge-robot committed:
Merge pull request #7377 from sanketpathak/gherkin_display_TaskRuns_associated_with_PipelineRun_ODC-3712

Gherkin display task runs associated with pipeline run odc 3712

rawagner committed:
i18n for dropdown object enums

bipuladh committed:
Update feature guards
Refactor loading of CSV and Infrastruture in Install Page

openshift-merge-robot committed:
Merge pull request #7406 from afreen23/i18n-ocs-day1and2

i18n ocs day1 and day2 operations

openshift-merge-robot committed:
Merge pull request #7412 from rohitkrai03/helm-plugin

Move helm from dev-console into its own monorepo package

rohitkrai03 committed:
Move gitops code out into its own monorepo package

christoph-jerolimov committed:
Get customized catalog categories in dev catalog from server flags

* Add server flags in bridge.
* Change (default) categories to new format (tags instead of field and values, arrays instead of records).
* Change frontend to show these categories from server flags.

openshift-merge-robot committed:
Merge pull request #7337 from pcbailey/localize-remaining-kubevirt-modals

Localize remaining kubevirt-plugin modals

openshift-merge-robot committed:
Merge pull request #7407 from sahil143/odc-5111-perspective

convert last used perspective local storage to user settings

rebeccaalpert committed:
Add tour file

pcbailey committed:
Localize more kubevirt strings

christoph-jerolimov committed:
Save dropdown bookmarks and favoriteKey in user settings

christianvogt committed:
Revert "remove tech preview badge from web terminal"

This reverts commit 7ef4fcaa8670a3100136f3b7c6a29a3a5e54bd52.

openshift-merge-robot committed:
Merge pull request #7354 from makambalaji/odc-5181-add-flow--container-image-scenarios

Cypress automation add flow container image scenarios - odc 5181

openshift-merge-robot committed:
Merge pull request #7382 from dtaylor113/cypress-a11y-aria-hidden-focus

Bug 1897008: Cypress: reenable check for 'aria-hidden-focus' rule & checkA11y test for modals

openshift-merge-robot committed:
Merge pull request #7393 from rhamilto/bz1897354

Bug 1897354:  make CRDCard display consistent with tabs

openshift-merge-robot committed:
Merge pull request #7401 from gajanan-more/enhanced-catalog

Gherkin Scripts for ODC-4181 and ODC-3954

openshift-merge-robot committed:
Merge pull request #7415 from bipuladh/fix_loading_capacity

Fix error state for Raw Capacity Card

openshift-merge-robot committed:
Merge pull request #7420 from rohitkrai03/gitops-plugin

Move gitops code out into its own monorepo package

spadgett committed:
Bug 1856803: Fix cluster monitoring link

cajieh committed:
Role name missing on create role binding form

jeff-phillips-18 committed:
Do not load topology resources when in all namespaces project is selected

openshift-merge-robot committed:
Merge pull request #7426 from spadgett/monitoring-link

Bug 1856803: Fix cluster monitoring link

openshift-merge-robot committed:
Merge pull request #7376 from cloudbehl/arbiter-cluster

Create stretch cluster with arbiter node.

openshift-merge-robot committed:
Merge pull request #7410 from jeff-phillips-18/topology-pipelines

Use extensions to add decorators to topology nodes

openshift-merge-robot committed:
Merge pull request #7418 from rawagner/sc-vm-snapshot

add sc detection to vm snapshots

openshift-merge-robot committed:
Merge pull request #7422 from rawagner/object_enum_i18n

i18n for dropdown object enums

openshift-merge-robot committed:
Merge pull request #7423 from pcbailey/localize-more-kubevirt-strings

Localize more kubevirt strings

openshift-merge-robot committed:
Merge pull request #7379 from jeff-phillips-18/nav-items-sort

Fix sorting of nav section items

openshift-merge-robot committed:
Merge pull request #7390 from jerolimov/odc-5111-dropdown

Save dropdown bookmarks and favoriteKey in user settings

openshift-merge-robot committed:
Merge pull request #7413 from jcaianirh/cm-dlist-background

Bug 1903172: Background shadow appears under datalist in column management modal

openshift-merge-robot committed:
Merge pull request #7330 from a2batic/storage-class-kms

KMS flow for storage class creation

openshift-merge-robot committed:
Merge pull request #7356 from makambalaji/odc-5182-add-flow-docker-flow-scenarios

Cypress Automation - Add flow docker flow scenarios-ODC-5182

openshift-merge-robot committed:
Merge pull request #7357 from makambalaji/odc-5184-catalog-file-scenarios

Cpress automation scripts for Add Flow catalog scenarios-odc-5184

openshift-merge-robot committed:
Merge pull request #7409 from sanketpathak/gherkin_usability_enhancement_for_quick_start-ODC-4653

gherkin_usability_enhancement_for_quick_start-ODC-4653

openshift-merge-robot committed:
Merge pull request #7425 from christianvogt/revert-terminal-badge

Revert "remove tech preview badge from web terminal"

openshift-merge-robot committed:
Merge pull request #7196 from jerolimov/odc-5019

Get customized catalog categories in dev catalog from server flags

openshift-merge-robot committed:
Merge pull request #7297 from jerolimov/odc-5080

Provide code snippet for default console catalog categories

openshift-merge-robot committed:
Merge pull request #7428 from jeff-phillips-18/topology-all-ns

Do not load topology resources when in all namespaces project is selected

vikram-raj committed:
Bug 1872166: Fix Silences link redirection from developer perspective

openshift-merge-robot committed:
Merge pull request #7416 from afreen23/feat-flexible-scaling

(feat): Enable flexible scaling in storage cluster deployment

openshift-merge-robot committed:
Merge pull request #7375 from makambalaji/odc-5188-create-pipelines-from-add-options

Cypress Automation - Pipelines for add options-odc-5188

glekner committed:
Fixes 1904776: Snapshot modal alert is not pluralized

openshift-merge-robot committed:
Merge pull request #7434 from glekner/fix-1904776

Bug 1904776: Snapshot modal alert is not pluralized

karthikjeeyar committed:
Fix pipelinerun visualization inprogress crash

metalice committed:
Namespace is passed differently when using create vm from template. added support for this url type

Signed-off-by: Matan Schatzman <matanschatzman@gmail.com>

karthikjeeyar committed:
Fix pipelinerun visualization inprogress crash

gajanan-more committed:
Gherkin Script Scenarios for epic ODC-4658

rottencandy committed:
hide scale buttons in donut for Pods

divyanshiGupta committed:
Do not show misleading alert message

glekner committed:
Fixes 1905104 - Snapshot modal disk items missing keys

openshift-merge-robot committed:
Merge pull request #7439 from metalice/bz-1902981-creating-vm-from-template-should-set-to-current-namespace

Bug 1902981: Namespace is passed differently when using create vm from template.

dtaylor113 committed:
Cypress: do not test for i18n translation of timestamps

openshift-merge-robot committed:
Merge pull request #7009 from jcaianirh/i18n-api-explorer

CONSOLE-2431: i18n for home/api-explorer

openshift-merge-robot committed:
Merge pull request #7038 from TheRealJon/1892198

Bug 1892198: Fix runtime error on operand details page.

rebeccaalpert committed:
Add revised files

rebeccaalpert committed:
Bug 1904993: Remove &amp; in strings

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1904993

openshift-merge-robot committed:
Merge pull request #7266 from kyoto/query-browser-i18n

CONSOLE-2391: Monitoring: Add i18n for Metrics and Dashboards pages

openshift-merge-robot committed:
Merge pull request #7445 from dtaylor113/cypress-do-not-test-i18n-for-timestamps

Cypress: do not test for i18n pseudo-translation of timestamps

openshift-merge-robot committed:
Merge pull request #7259 from harishsurf/sources-tab

CONSOLE-2477: Improve OperatorHub -> Sources view

openshift-merge-robot committed:
Merge pull request #7442 from glekner/fix-1905104

Bug 1905104: Snapshot modal disk items missing keys

openshift-merge-robot committed:
Merge pull request #7248 from vikram-raj/odc-4901

Bug 1872166: Fix Silences link redirection from developer perspective

gajanan-more committed:
Gherkin Scripts Scenarios epic ODC-1052

invincibleJai committed:
route eventsource from Admin to catalog for source and workload on create

rawagner committed:
Default to RedHat/KubeVirt provider if common template does not have provider annotation

Commit:
Bug 1904244: Drop using i18next.t from plugin.ts file

Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1904244

Signed-off-by: Afreen Rahman <afrahman@redhat.com>

akashshinde committed:
Fix unit test failure

openshift-merge-robot committed:
Merge pull request #7433 from sahil143/odc-5111-namespace-2

convert last used namespace local storage to userSettings

glekner committed:
Fixes 1881268: Image uploading failed but wizard claim the source is available

rawagner committed:
Show error if common template does not have SRC_PV params

abhinandan13jan committed:
taskBar width

openshift-merge-robot committed:
Merge pull request #6314 from glekner/test-snapshots

Bug 1882268: Add snapshots integration test

openshift-merge-robot committed:
Merge pull request #7449 from rebeccaalpert/amp

Bug 1904993: Remove &amp; in strings

pcbailey committed:
Add selector for active users on Overview tab

ashishkumsingh committed:
Replace 'Baremetal' with 'BareMetal' in the supported platforms for
Independent mode of OCS deployment.

Signed-off-by: Ashish Singh <assingh@redhat.com>

openshift-merge-robot committed:
Merge pull request #7452 from invincibleJai/odc-5220

route eventsource from Admin to catalog for source and workload on create

rohitkrai03 committed:
Use chart repository spec name when available

rohitkrai03 committed:
Show type badge on every tile in all catalogs

openshift-merge-robot committed:
Merge pull request #7224 from rebeccaalpert/new-translations

CONSOLE-2475: Check in Japanese and Chinese translations

openshift-merge-robot committed:
Merge pull request #7444 from rottencandy/podscale

Hide scale buttons of pod donut for Pod resources

openshift-merge-robot committed:
Merge pull request #7269 from rawagner/vmt_check_cert

Check certificate before uploading

openshift-merge-robot committed:
Merge pull request #7454 from afreen23/bug-fix-missing-keys

Bug 1904244: Drop usage of i18next.t from plugin.ts files

openshift-merge-robot committed:
Merge pull request #7455 from akashshinde/fix-unit-tests

 Bug 1905146: Fix helm unit test failures

openshift-merge-robot committed:
Merge pull request #7463 from ashishkumsingh/fix_indep_mode_on_bm_deployment

Bug 1905542: Replace 'Baremetal' with 'BareMetal' in the supported platforms for I…

openshift-merge-robot committed:
Merge pull request #7468 from rohitkrai03/catalog-label-fix

Show type badge on every tile in all catalogs

openshift-merge-robot committed:
Merge pull request #7427 from cyril-ui-developer/role-name-missing-create-role-binding-form

Bug 1903192: - Role name missing on create role binding form

openshift-merge-robot committed:
Merge pull request #7460 from abhinandan13jan/task-width

add minwidth to taskBar

akashshinde committed:
Added doc for disabling helm chart repository

glekner committed:
Bug 1879419: Improve boot source description for Container and URL

openshift-merge-robot committed:
Merge pull request #7458 from glekner/fix-1881268

Bug 1881268: Image uploading failed but wizard claim the source

glekner committed:
Bug 1879013: Improve CD-ROM interface selection

rawagner committed:
Add missing OS name to details popover.
Show 'Not available' if there's no workload profile

bipuladh committed:
Label namespace during storage cluster creation

openshift-merge-robot committed:
Merge pull request #7419 from bipuladh/feature_gurads

Add feature guard for Multus

openshift-merge-robot committed:
Merge pull request #7453 from rawagner/vmt_provider

Bug 1905380: Default to Red Hat/KubeVirt provider if common template does not have …

bond95 committed:
Fixed endless loading state for insights plugin

christianvogt committed:
fix quick start icon tile alignment

openshift-merge-robot committed:
Merge pull request #7443 from divyanshiGupta/ODC-3289

Fix to not show misleading alert message in Project Access Page

openshift-merge-robot committed:
Merge pull request #7459 from rawagner/vmt_missing_base

Show error if common template does not have SRC_PV params

openshift-merge-robot committed:
Merge pull request #7479 from glekner/fix-1879419

Bug 1879419: Improve boot source description for Container and URL

sg00dwin committed:
Remove border rule since upstream fix has merged
https://github.com/patternfly/patternfly/pull/2963

harishsurf committed:
Bug 1905941: Add optional chaining to catalog source Status field

Signed-off-by: Harish <hgovinda@redhat.com>

openshift-merge-robot committed:
Merge pull request #6873 from jerolimov/odc-4906

Improve secret hostname hint

openshift-merge-robot committed:
Merge pull request #7052 from vbnrh/default-snapshot-class-selection

Adds default label to snapshot class with the correct annotation

openshift-merge-robot committed:
Merge pull request #7462 from pcbailey/add-selector-for-active-users-on-overview-tab

Bug 1900377: Add selector for active users on Overview tab

openshift-merge-robot committed:
Merge pull request #7280 from zherman0/update-yarn

CONSOLE-2399: Update yarn to latest stable 1.22.10

openshift-merge-robot committed:
Merge pull request #7438 from karthikjeeyar/fix-pipelinerun-viz

Fix pipelinerun visualization inprogress crash

openshift-merge-robot committed:
Merge pull request #7448 from dtaylor113/cypress-annotations-test

Cypress Annotations tests

openshift-merge-robot committed:
Merge pull request #7491 from bond95/fixed-insights-loading

Fixed endless loading state for insights plugin

openshift-merge-robot committed:
Merge pull request #7493 from sg00dwin/remove-rule-fixed-upstream

Remove border rule since upstream fix has merged

openshift-merge-robot committed:
Merge pull request #7170 from rawagner/vm_templates_sc

Use kubevirt sc config map

openshift-merge-robot committed:
Merge pull request #7441 from gajanan-more/kamelets-eventsource

Bug 1905348: Gherkin Script Scenarios for epic ODC-4658

dankenigsberg committed:
kubevirt: use proper term for Authorized SSH Keys

cloud-init's ssh_authorized_keys should be called that way in code and
GUI because their important function is to AUTHORIZE a user to log into
a VM, not mere authentication. I suspect that the existing term was
introduced by mistake.

Signed-off-by: Dan Kenigsberg <danken@redhat.com>

bipuladh committed:
Trim user provided name for OBC before attempting creation

Commit:
Externalize the recent chnages on kms and flexible scaling
- strings of kms storage class form and fexible scaling were added later and were not marked for i18n

Signed-off-by: Afreen Rahman <afrahman@redhat.com>

openshift-merge-robot committed:
Merge pull request #7485 from rawagner/os_details_popover

Add missing OS name to details popover.

openshift-merge-robot committed:
Merge pull request #7492 from christianvogt/qs-tile-icon

fix quick start catalog tile icon alignment

vikram-raj committed:
Fix QuickStartPanelContent style

rhamilto committed:
Bug 1906186: use 'the' instead of 'this/these' in alert message

openshift-merge-robot committed:
Merge pull request #7487 from bipuladh/label_ns

Bug 1866298: Label namespace during storage cluster creation

openshift-merge-robot committed:
Merge pull request #7503 from dankenigsberg/authorized_keys

Bug 1906318: kubevirt: use proper term for Authorized SSH Keys

rhamilto committed:
Bug 1906441:  fix keys for HorizontalNav and NavBar

openshift-merge-robot committed:
Merge pull request #7483 from glekner/fix-1879013

Bug 1879013: Improve CD-ROM interface selection

openshift-merge-robot committed:
Merge pull request #7504 from bipuladh/fix_trailing_whitespace

Bug 1888544: Trim user provided name for OBC before attempting creation

openshift-merge-robot committed:
Merge pull request #7506 from afreen23/i18n-kms-flexible-scaling

Externalize the recent changes on kms and flexible scaling

openshift-merge-robot committed:
Merge pull request #7494 from harishsurf/catsrc-undefined

Bug 1905941: Add optional chaining to catalog source Status field

openshift-merge-robot committed:
Merge pull request #7510 from rhamilto/bz1906186

Bug 1906186: use 'the' instead of 'this/these' in alert message

openshift-merge-robot committed:
Merge pull request #7514 from rhamilto/bz1906441

Bug 1906441:  fix keys for HorizontalNav and NavBar

invincibleJai committed:
fixes issue with kn resources if trigger goes to ksvc and IMC from same broker

openshift-merge-robot committed:
Merge pull request #7451 from gajanan-more/import-devfile

Bug 1905347: Gherkin Scripts Scenarios epic ODC-1052

openshift-merge-robot committed:
Merge pull request #7508 from vikram-raj/odc-5190

Bug 1906679: Fix QuickStartPanelContent style

openshift-merge-robot committed:
Merge pull request #7509 from akashshinde/add-helm-disabled-repo-docs

Bug 1906691: Added doc for disabling helm chart repository

openshift-merge-robot committed:
Merge pull request #7519 from invincibleJai/odc-5254

Bug 1906683: fixes issue with kn resources if trigger goes to ksvc and IMC from same broker

openshift-merge-robot committed:
Merge pull request #7466 from rohitkrai03/helm-repo-name-fix

Bug 1906718: Use chart repository spec name when available

karthikjeeyar committed:
Fix Pipleine creation without namespace issue
